src/ruby/puppetserver-lib/puppet/server/master.rb

```diff
@@ -102,8 +111,14 @@ def self.getModules(env)
       # NOTE: If in the future we want to collect more
       #       Module settings, this should be more programatic
       #       rather than getting these settings one by one
-      module_data[:name] = mod.forge_name
-      module_data[:version] = mod.version
+      if mod.forge_name
+        module_data[:name] = mod.forge_name
+        module_data[:version] = mod.version
```

Comment:
Thanks, @briancain.  Given that in this scenario we'd probably see failures outside of the use of this API anyway, I'm inclined not to worry about this part further for now, then.  Thanks for checking into it!